### ISSUE TITLE
Use more varied accent colors

### ISSUE THREAD

**Issue by kimberly990** [joeyballentine/chaiNNer#664]
Certain node categories have accents that are far too similar, leading to initial confusion when relying on them to differentiate categories. This applies to the Image (Dimensions), Image (Utilities), and ONNX categories specifically.

**Comment by joeyballentine** [comment]
The caveat to this is we will eventually run out of accent colors. If we need to be super differentiable for each, that leaves us with very few colors we are able to use for categories. While now yeah we could easily just swap over something like Image Adjustment to purple or something, then what? We already use red, orange, yellow, green, blue, teal, pink, and gray.

I think ultimately the problem here is that the category colors we have are kinda meaningless. We really only color things differently because it looks cooler that way, not because it truly provides utility. And the little utility it does provide will be void if we add any more categories than we already have since we'll have to use more "close together" colors.

So I'm not really sure what to do about this.

**Comment by RunDevelopment** [comment]
Are category colors that important? What vital information do you get from knowing (or guessing) the category of a node from its accent color?

**Comment by kimberly990** [comment]
I disagree that they're meaningless. I know you two don't rely on them, but it makes using chaiNNer far more natural for me. I'm sure other users rely on it too to some extent.

I understand we just don't have a lot of color options unfortunately. I'd rather keep the accents in their current form than have them removed though.

**Comment by UNKNOWND3v** [comment]
What about allowing the user to change UI colours ?

**Comment by theflyingzamboni** [comment]
I have to agree with @kimberly990, having the different categories have different colors makes everything much more visually parseable for me. Removing them would make it harder for me to quickly pick a node out of a large chain. Don't really know what to do about limited color possibilities, but I do think having category colors matters for usability. It's definitely not meaningless.
